graphene_django/utils/testing.py

```diff
@@ -99,10 +94,19 @@ def query(self, query, op_name=None, input_data=None, variables=None, headers=No
             input_data=input_data,
             variables=variables,
             headers=headers,
-            client=self._client,
+            client=self.client,
             graphql_url=self.GRAPHQL_URL,
         )
 
+    @property
+    def _client(self):
```

Comment:
Assuming people calling (get attribute) the client with `_client` in their tests, including myself, i didn't want to break all at once. But i'm not sure how to handle set attribute case, or does it even necessary.

Reply:
Sensible.